app/src/components/main/modal/modalLayout/modalField/modalField.jsx

```diff
@@ -4,43 +4,51 @@ import styles from './modalField.scss';
 
 const cx = classname.bind(styles);
 
-export const ModalField = ({ label, children }) => (
+export const ModalField = ({ label, children, labelWidth, inputWidth  }) => (
```

Comment:
Do we really need to add additional prop inputWidth? As I know, label & input are flexible and if we set a hardcoded value to labelWidth, input will be stretched to full available space. Just set up ModalField width and label width.